packages/rancher-gatekeeper/generated-changes/exclude/templates/crds.yaml

```diff
@@ -1,6 +0,0 @@
-{{- if .Values.customResourceDefinitions.create }}
```

Comment:
seems like this no longer exists in chart. LGTM

Reply:
This file is removed from upstream v3.4.0 onwards